src/js/button.js

```diff
@@ -7,13 +12,13 @@
  * @class
  * @constructor
  */
-vjs.Button = vjs.Component.extend({
+var Button = Component.extend({
   /**
    * @constructor
    * @inheritDoc
    */
   init: function(player, options){
```

Comment:
Yeah, I like that. I just need to get more familiar with how the migrate plugin would work. Like would it be throwing deprecation warnings for older APIs or just silently make them available again?